packages/react/src/components/icon/icon.stories.tsx

```diff
@@ -1,35 +1,30 @@
-import { Color } from '@onfido/castor';
 import { iconNames } from '@onfido/castor-icons';
 import { Icon, IconProps } from '@onfido/castor-react';
 import React from 'react';
-import { Meta, omit, Story, storyOf, tokens } from '../../../../../docs';
-
-const colors = Object.keys(tokens).reduce((accumulator, name) => {
-  const [, color] = name.match(/^--ods-color-([a-z0-9-]+)$/) ?? [];
-  return color ? [...accumulator, color] : accumulator;
-}, []) as Color[];
+import { colors, Meta, omit, reactMatrix, Story } from '../../../../../docs';
 
 const [firstIconName] = iconNames;
 
 export default {
   title: 'React/Icon',
   component: Icon,
   argTypes: {
-    ...omit<IconProps>('className', 'style'),
-    name: { control: { type: 'select', options: iconNames } },
```

Comment:
Curious, so "name" now gets rendered based on type right? What about "color", why that does not work then?

Reply:
Correct, all the possible name values are picked up by storybook/react-docgen.
I don't know why it doesn't work for color, probably because the types are imported from another library?

Reply:
OK, cool, no worries then - it can be explicitly set here.